job-processor2/src/test/java/dk/dbc/dataio/jobprocessor2/util/FlowCacheTest.java

```diff
@@ -1,107 +0,0 @@
-package dk.dbc.dataio.jobprocessor2.util;
```

Comment:
Vi skal have et minimum antal tests på FlowCache, så vi opdager hvis nogen har snublet over tastaturet :) 
Det behøver ikke være noget fancy, bare findes-findes ikke for et item - men vi skal have dækket koden.

Reply:
Så den cache der ligger bag, er nu en standard google cache, og der er absolut ingen grund til at vi tester om noget der er blevet puttet ind i cachen, også kan komme ud igen... Det har de klaret for os, derfor slettede jeg de eksisterende test.